<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myclass</class>
 <widget class="QMainWindow" name="myclass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="classtableWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>631</width>
      <height>251</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>周一</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>周二</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>周三</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>周四</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>周五</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>周六</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>周日</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>第一节</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>第二节</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>第三节</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>第四节</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>第五节</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>第六节</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>第七节</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>第八节</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>第九节</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>第十节</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>第十一节</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>第十二节</string>
     </property>
    </column>
   </widget>
   <widget class="QListWidget" name="finallistWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>320</y>
      <width>491</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addpushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>400</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deletepushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>460</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QListWidget" name="classlistWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>51</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
